x-pack/plugins/ml/server/routes/json_schema.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getJsonSchemaQuerySchema } from './schemas/json_schema_schema';
+import { wrapError } from '../client/error_wrapper';
+import { RouteInitialization } from '../types';
+import { JsonSchemaService } from '../models/json_schema_service';
+
+export function jsonSchemaRoutes({ router, routeGuard }: RouteInitialization) {
```

Comment:
`apidoc.json` needs updating with these routes.

Reply:
Updated in https://github.com/elastic/kibana/pull/146968/commits/d394d48f3e5fbb3d4d6b4fb5adc3a79cda9eb186